makezip.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o errexit -o nounset
+
+MY_MODULES=$(git submodule status | awk '{print $2}')
+
+MY_PYTHON=$(command -v python3)
+
+
+zipfile=$PWD/modules.zip
+
+rm -f "$zipfile"
+
+for m in $MY_MODULES; do
+    echo "=== Zipping $m"
+    cd "$m"
+    zip -r "$zipfile" "$m/"
+    cd ..
+done
+
+
+echo "=== Preparing path file of '$MY_PYTHON'"
+echo "=== for importing modules from '$zipfile'"
+echo
+
+sitefile="$($MY_PYTHON -c 'import site; print(site.getsitepackages()[0])')/emirge.pth"
+
+echo "$zipfile" > "$sitefile"
+
+echo "=== Done. Make sure to uninstall other copies of the emirge modules and"
```

Comment:
> Make sure to uninstall other copies of the emirge modules

I think this comment is kind of unhelpful. I wouldn't trust myself to do this correctly. How will users accomplish this?

Reply:
I added more details to the echo here. The reason is that if a module is also installed via the normal `pip` commands, that pipp'ed module seems to have preference over the modules.zip module.